src/app/util/attribute-storage.cpp

```diff
@@ -129,7 +129,7 @@ static uint16_t findClusterEndpointIndex(EndpointId endpoint, ClusterId clusterI
 // Initial configuration
 void emberAfEndpointConfigure(void)
 {
-    uint8_t ep;
+    uint16_t ep;
```

Comment:
Can you update `kTestEndpointId` and `kTestEndpointId3` in `TestReadChunking.cpp` to occupy values in the 16-bit space? That should give us unit-test coverage.

Please ensure that the test fails before this change.

Reply:
Nvm. There is no code-path with dynamic endpoints that ends up affecting this logic here in `emberAfEndpointConfigure`.

Reply:
Short of actually modifying the example apps to have endpoints in this space, there's no way to test this in unit-test :(